app/controllers/concerns/whitelisted_flow_step_concern.rb

```diff
@@ -0,0 +1,37 @@
+# This Concern satisfies the brakeman gem "Dynamic Render Path" violation
+# that is raised when rendering dynamic content in views and partials
+# that come directly from params. In the below example, idv_inherited_proofing_cancel_path
+# would render "/verify/inherited_proofing/cancel?step=<params[:step]>" where <params[:step]>
+# == the value of params[:step], which could potentially be dangerous:
+# <%= render ButtonComponent.new(action: ->(...) do
+#     button_to(idv_inherited_proofing_cancel_path(step: params[:step]), ...) ...
+#     end
+# %>
+module WhitelistedFlowStepConcern
```

Comment:
I'd suggest renaming this to `AllowlistedFlowStepConcern` as a more inclusive name.

Reply:
Will do. I assume you meant `AllowWhitelistedFlowStepConcern`

Reply:
I believe that @mitchellhenke was accurate in suggesting `AllowlistedFlowStepConcern`, since there's a preference to avoid the term "whitelist" in favor of "allowlist" in the code.

Reply:
Ah ok, I'll change that in the next branch.